### ISSUE TITLE
unused-variable emitted instead of unused-import

### ISSUE THREAD

**Issue by sushobhit27** [PyCQA/pylint#2421]
No

### Steps to reproduce
1. cat sample6.py
```
def foobar():
    from functools import wraps
```
2. pylint sample6.py

### Current behavior
************* Module sample6
sample6.py:2:4: W0612: Unused variable 'wraps' (unused-variable)

------------------------------------------------------------------
Your code has been rated at 5.00/10 (previous run: 0.00/10, +5.00)

### Expected behavior
Instead of unused-variable, unused-import should be raised.

The behaviour is correct when import statement is at top. e.g
```
from functools import wraps

def foobar():
    pass

```
pylint sample6.py

************* Module sample6
sample6.py:1:0: W0611: Unused wraps imported from functools (unused-import)

------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 5.00/10, +1.67)

### pylint --version output
pylint 2.2.0
astroid 2.0.1
Python 3.6.4 (default, Jan  7 2018, 15:53:53)
[GCC 6.4.0]
